src/components/sbb-teaser-hero/sbb-teaser-hero.tsx

```diff
@@ -110,24 +56,17 @@ export class SbbTeaserHero {
     }
 
     return (
-      <a class="teaser-hero" href={this.link} {...linkAttributes}>
-        <sbb-image
-          class="teaser-hero__image"
-          pictureSizesConfig={JSON.stringify(this._pictureSizesConfig)}
-          customFocalPoint={true}
-          hideFromScreenreader={true}
-          imageSrc={this.imageSrc}
-          loading={this.imageLoading}
-          lqip
-          performanceMark="teaser-hero"
-          variant="teaser-hero"
-        ></sbb-image>
-
-        <sbb-panel
-          class="teaser-hero__panel"
-          buttonText={this.buttonText}
-          text={this.text}
-        ></sbb-panel>
+      <a
+        class="teaser-hero"
+        href={this.link}
+        aria-label={this.accessibilityTitle}
+        {...linkAttributes}
+      >
+        <sbb-title level="1" visually-hidden="true" text={this.accessibilityTitle} />
```

Comment:
I feel that I already asked that, but is the sbb-title not an overkill here? It also renders headline tags which are block elements and therefore not optimized to use inside a link

Reply:
tha aria label already shows the accessibilityLabel